numpyro/infer/util.py

```diff
@@ -870,6 +882,9 @@ class Predictive(object):
 
         + set `batch_ndims=1` to get predictions from a one dimensional batch of the guide and parameters
           with shapes `(num_samples x batch_size x ...)`
+    :param exclude_deterministic_params: indicates whether deterministic sites should get sampled
```

Comment:
maybe just `exclude_deterministic`? `params` are parameters of the model. The description could be: indicates whether to ignore deterministic sites from the posterior samples.